pkgs/unified_analytics/USAGE_GUIDE.md

```diff
@@ -94,17 +94,29 @@ of the tool using this package to display the proper Analytics messaging
 and inform them on how to Opt-Out of Analytics collection if they wish. The
 package will expose APIs that will make it easy to configure Opt-In status.
 
-```dart
-// Begin by initializing the class
-final Analytics analytics = Analytics(...);
+For this use case, it is best to use the `Analytics` static method `getConsentMessage`
+which requires passing in the `DashTool` currently using this package
 
-// This should be performed every time the tool starts up
-if (analytics.shouldShowMessage) {
+Passing in the `DashTool` is necessary so that the message can include which
+tool is seeking consent
 
-  // How each tool displays the message will be unique,
-  // print statement used for trivial usage example
-  print(analytics.toolsMessage);
-}
+```dart
+final String consentMessage = Analytics.getConsentMessage(tool: DashTool.flutterTools);
+
+/// Printing out the consent message above returns the below
+///
+/// The flutter_tools uses Google Analytics to report usage and diagnostic data
```

Comment:
It looks weird to have the underscore. Could we replace it with "flutter tools"?

Reply:
Yep, i was printing out the label we used for `DashTool` but it should be pretty easy to do a global replace of all underscores with spaces when returning the consent message